SimpleCalculator.py

```diff
@@ -71,32 +72,43 @@ def find_min_value(values):
 # TODO this function should take list of values, then return value with biggest value
 def find_max_value(values):
     max_value = 0
-    # TODO  here should be your implementation
+
+
     return max_value
 
 
 # TODO this function should take list of values, then return average of those values
 
+
 def calculate_average(values):
     average = 0
     average = sum(values) / len(values)
-    # TODO  here should be your implementation
     return average
 
 
 
 # TODO this function should take list of values,
 # then returns sorted list in ascending(normal) or descending(reverse" order
 def sort_values(values, order="normal"):
```

Comment:
ok, nice almost works :D if you check test for this function, then you should still have some failing test, to run those test run TestSortingFunction function